Gopkg.toml

```diff
@@ -557,3 +557,7 @@ required = [
     name = "k8s.io/code-generator"
     unused-packages = false
     non-go = false
+
+[[constraint]]
+  branch = "master"
```

Comment:
to avoid having this changing every time someone runs `dep ensure -v` pick the latest commit from the master branch. Also, keep this file in alphabetical order. This should be around line 456